app/components/gh-file-uploader.js

```diff
@@ -21,7 +21,8 @@ export default Component.extend({
     labelText: 'Select or drag-and-drop a file',
     url: null,
     paramName: 'file',
-    accept: 'text/csv',
+    accept: ['text/csv'],
+    extensions: ['csv'],
```

Comment:
Maybe we can use `didReceiveAttrs` to build an internal array from any supplied strings?